src/parser/hunter/beastmastery/modules/Abilities.tsx

```diff
@@ -29,7 +29,7 @@ class Abilities extends CoreAbilities {
       {
         spell: SPELLS.KILL_COMMAND_CAST_BM,
         category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
-        cooldown: haste => 7.5 / (1 + haste),
+        cooldown: (haste: number) => 7.5 / (1 + haste),
```

Comment:
If you look at the Holy Paladin version you can see that this can be omitted if you manually set `spellbook`'s return type to `SpellbookAbility[]`.